docs/software/jupyter-notebook.md

```diff
@@ -10,7 +10,7 @@ exclude: true
 
 You can run a Jupyter Lab server as a cluster job and connect to it using a web browser on your local computer. This setup is very convenient for interacting with large data sets and computational expensive operations that would grind your laptop to a halt. It's also a great way to incrementally develop batch jobs to run on the cluster.
 
-## Setting Up a Worker Node
+## Setting Up a Computer Node
```

Comment:
We should make this consistent so suggest `s/Computer/Compute/`

Reply:
Curse my slippery fingers!  Force-pushed `Computer` --> `Compute`.